scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
As we have limitation of building dockerfile only on targeted platform and can't be cross-built on different platform as of now. So this will not create any issues I feel because I'm using this only while building docker image.